basemq/CMakeLists.txt

```diff
@@ -11,22 +11,14 @@
 set(target BaseMQ)
 
 list(APPEND headers
-  baseMQtools/baseMQtools.h
   devices/FairMQLmdSampler.h
   devices/FairMQProcessor.h
   devices/FairMQSampler.h
   devices/FairMQUnpacker.h
-  policies/Sampler/FairMQFileSource.h
-  policies/Sampler/FairSourceMQInterface.h
-  policies/Sampler/SimpleTreeReader.h
-  policies/Serialization/BinaryBaseClassSerializer.h
```

Comment:
Could you please write a ChangeLog-Entry for the removal part? It's really a breaking change in my view. Even if nobody known to us uses it, I would prefer proper ChangeLogs for this kind of things.

Reply:
Done.